lib/Server.js

```diff
@@ -264,24 +265,41 @@ Server.prototype.setContentHeaders = function(req, res, next) {
 	next();
 }
 
-// delegate listen call and init socket.io
+// delegate listen call and init sockjs
 Server.prototype.listen = function() {
 	this.listeningApp.listen.apply(this.listeningApp, arguments);
-	this.io = socketio.listen(this.listeningApp, {
-		"log level": 1
+	var sockServer = sockjs.createServer({
+		// Limit useless logs
+		log: function(severity, line) {
+			if (severity === 'error') {
+				console.log(line);
+			}
+		}
 	});
-	this.io.sockets.on("connection", function(socket) {
-		if(this.hot) socket.emit("hot");
+	sockServer.on("connection", function(conn) {
+		this.sock = conn;
```

Comment:
Wow, that was a pretty big oversight of me. Pretty ashamed I didn't notice that myself. Anyway, it should be fixed now.